docs/reference/filebeat/delete-file-guide.md

```diff
@@ -0,0 +1,171 @@
+# Removing files after ingestion
+
+::::{warning}
+Enabling this feature will remove files, which could lead to unintentional data loss if not configured correctly.
+::::
+
+The Filestream input can remove files after they have been fully
```

Comment:
Consider adding a substitution here for `Filestream`, such as `fs`: https://github.com/elastic/beats/blob/main/docs/docset.yml

Reply:
> This feature is always on by default then?

No, it is off by default.

> Shouldn't we start by stating what's the default state and how to control it?

It kinda was like that, the very first sentence was:
```
Enabling this feature will remove files, which could lead to unintentional data loss if not configured correctly.
```

But then you suggested updating it to:
```
This feature might lead to unintentional data loss if not configured correctly.
```

I'm fine either way. To me it is very clear it's disabled by default because later we show that to enable the feature we need to set `delete.enabled: true`. But I wrote the whole thing, so I'm biased when comes to understanding how it works.

Reply:
I suggest a start like this: "By default, Filestream input doesn't delete files. If **option** is turned on, Filestream input can delete files under these conditions... "